scss/window/_layout.scss

```diff
@@ -21,19 +21,23 @@
 
     // Title bar
     .k-window-titlebar {
-        padding: $window-titlebar-padding-y $window-titlebar-padding-x;
-        min-height: 16px;
-        border-width: 0 0 1px;
-        border-style: solid;
```

Comment:
I was wondering the need for the min-height as well - it wouldn't be required.

As per the borders - I haven't removed the default styling but rather just moved them into variables so that they could be customized if required.

To explain the reasoning behind the shorthand padding I have included a screenshot of the outcome of these variables.

![copperleaf-headerbar-design](https://user-images.githubusercontent.com/884395/27920788-93fe63da-622a-11e7-8b1d-d8f327c149c2.png)